--- a/install/controllers/http/process.php
+++ b/install/controllers/http/process.php
@@ -100,12 +100,8 @@ class InstallControllerHttpProcess extends InstallControllerHttp implements Http
                 $this->processConfigureShop();
             } elseif (Tools::getValue('installModules') && (!empty($this->session->process_validated['configureShop']) || $this->session->install_type != 'full')) {
                 $this->processInstallModules();
-            } elseif (Tools::getValue('installModulesAddons') && !empty($this->session->process_validated['installModules'])) {
-                $this->processInstallAddonsModules();
-            } elseif (Tools::getValue('installTheme') && !empty($this->session->process_validated['installModulesAddons'])) {
+            } elseif (Tools::getValue('installTheme') && !empty($this->session->process_validated['installModules'])) {
                 $this->processInstallTheme();
-            } elseif (Tools::getValue('installFixtures') && !empty($this->session->process_validated['installTheme'])) {
-                $this->processInstallFixtures();
             }
         } catch (\Exception $e) {
             $this->ajaxJsonAnswer(false, $e->getMessage());
@@ -303,10 +299,13 @@ class InstallControllerHttpProcess extends InstallControllerHttp implements Http
         $this->process_steps[] = array('key' => 'configureShop', 'lang' => $this->translator->trans('Configure shop information', array(), 'Install'));
 
         $this->process_steps[] = array('key' => 'installModules', 'lang' => $this->translator->trans('Install modules', array(), 'Install'));
+/*
         $this->process_steps[] = array('key' => 'installModulesAddons', 'lang' => $this->translator->trans('Install Addons modules', array(), 'Install'));
+*/
 
         $this->process_steps[] = array('key' => 'installTheme', 'lang' => $this->translator->trans('Install theme', array(), 'Install'));
 
+/*
         if ($this->session->install_type == 'full') {
             $fixtures_step = array('key' => 'installFixtures', 'lang' => $this->translator->trans('Install demonstration data', array(), 'Install'));
             if ($this->hasLargeFixtures()) {
@@ -321,6 +320,7 @@ class InstallControllerHttpProcess extends InstallControllerHttp implements Http
             }
             $this->process_steps[] = $fixtures_step;
         }
+*/
 
         $this->displayTemplate('process');
     }
