diff --git a/install/theme/views/header.phtml b/install/theme/views/header.phtml
index 0bf2565..0cb1c6e 100644
--- a/install/theme/views/header.phtml
+++ b/install/theme/views/header.phtml
@@ -35,13 +35,6 @@
 		<li class="lnk_forum"><a href="<?php echo $this->getSupportLink() ?>" target="_blank"><?php echo $this->l('Support') ?></a></li>
 		<li class="lnk_forum"><a href="<?php echo $this->getDocumentationLink() ?>" target="_blank"><?php echo $this->l('Documentation') ?></a></li>
 		<li class="lnk_blog last"><a href="<?php echo $this->getBlogLink() ?>" target="_blank"><?php echo $this->l('Blog') ?></a></li>
-		<!--
-		<?php if ($this->getPhone()): ?>
-			<li id="phone_block" class="last">
-				<div><span><?php echo $this->l('Contact us!') ?></span><br /><?php echo $this->getPhone() ?></div>
-			</li>
-		<?php endif; ?>
-		-->
 	</ul>
 
 	<div id="PrestaShopLogo">PrestaShop</div>
@@ -84,7 +77,6 @@
 	<?php if (@fsockopen('api.prestashop.com', 80, $errno, $errst, 3)): ?>
 		<iframe scrolling="no" style="height:210px;width:200px;border:none;margin-top:20px" id="iframe_help"
 			src="https://api.prestashop.com/iframe/install.php?step=<?php echo $this->step ?>&lang=<?php echo $this->language->getLanguageIso() ?><?php if (isset($this->session->shop_country)) echo '&country='.$this->session->shop_country; ?>">
-			<p><?php echo $this->l('Contact us!') ?><br /><?php echo $this->getPhone() ?></p>
 		</iframe>
 	<?php endif; ?>
 </div>
