--- a/install/theme/views/header.php
+++ b/install/theme/views/header.php
@@ -19,7 +19,6 @@
     <script type="text/javascript" src="../js/jquery/jquery-migrate-3.1.0.min.js"></script>
     <script type="text/javascript" src="../js/jquery/plugins/jquery.chosen.js"></script>
 	<script type="text/javascript" src="theme/js/install.js"></script>
-	<script type="text/javascript" src="https://www.prestashop.com/js/user-assistance.js"></script>
 	<?php if (file_exists(_PS_INSTALL_PATH_.'theme/js/'.self::$steps->current()->getName().'.js')): ?>
 		<script type="text/javascript" src="theme/js/<?php echo self::$steps->current()->getName() ?>.js"></script>
 	<?php endif; ?>
