diff --git a/install/controllers/http/welcome.php b/install/controllers/http/welcome.php
index f5d6b7f..8af0754 100644
--- a/install/controllers/http/welcome.php
+++ b/install/controllers/http/welcome.php
@@ -54,7 +54,7 @@ class InstallControllerHttpWelcome extends InstallControllerHttp implements Http
         if (!empty($this->session->lang) && !is_file(_PS_ROOT_DIR_.'/app/Resources/translations/'.$locale.'/Install.'.$locale.'.xlf')) {
             Language::downloadAndInstallLanguagePack($this->session->lang, _PS_VERSION_, null, false);
             $this->clearCache();
-            $this->redirect('welcome');
+//            $this->redirect('welcome');
         }
     }
 
