diff --git a/install/classes/datas.php b/install/classes/datas.php
index c937aa6..e342f40 100644
--- a/install/classes/datas.php
+++ b/install/classes/datas.php
@@ -148,7 +148,7 @@ class Datas
         ),
         'newsletter' => array(
             'name' => 'newsletter',
-            'default' => 1,
+            'default' => 0,
             'help' => 'get news from PrestaShop',
         ),
         'send_email' => array(
diff --git a/install/controllers/console/process.php b/install/controllers/console/process.php
index 56bf648..6284ba5 100644
--- a/install/controllers/console/process.php
+++ b/install/controllers/console/process.php
@@ -233,7 +233,7 @@ class InstallControllerConsoleProcess extends InstallControllerConsole
             'admin_password' =>            $this->datas->admin_password,
             'admin_email' =>            $this->datas->admin_email,
             'configuration_agrement' =>    true,
-            'send_informations' => true,
+            'send_informations' => false,
         ));
     }
 
diff --git a/install/controllers/http/configure.php b/install/controllers/http/configure.php
index 4707871..9448743 100644
--- a/install/controllers/http/configure.php
+++ b/install/controllers/http/configure.php
@@ -48,7 +48,7 @@ class InstallControllerHttpConfigure extends InstallControllerHttp
             $this->session->admin_firstname = trim(Tools::getValue('admin_firstname'));
             $this->session->admin_lastname = trim(Tools::getValue('admin_lastname'));
             $this->session->admin_email = trim(Tools::getValue('admin_email'));
-            $this->session->send_informations = Tools::getValue('send_informations');
+            $this->session->send_informations = false;
             if ($this->session->send_informations) {
                 $params = http_build_query(array(
                     'email' => $this->session->admin_email,
