diff --git a/install/controllers/http/welcome.php b/install/controllers/http/welcome.php
index 4055c720..7243398a 100644
--- a/install/controllers/http/welcome.php
+++ b/install/controllers/http/welcome.php
@@ -54,7 +54,7 @@ class InstallControllerHttpWelcome extends InstallControllerHttp implements Http
             $this->clearCache();
         }
         if (Tools::getIsset('language') && is_dir(_PS_ROOT_DIR_ . '/app/Resources/translations/' . $locale)) {
-            $this->redirect('welcome');
+//            $this->redirect('welcome');
         }
     }
 
