diff --git a/admin/themes/default/template/controllers/themes/helpers/options/options.tpl b/admin/themes/default/template/controllers/themes/helpers/options/options.tpl
index fc98b8f..7fa1a18 100644
--- a/admin/themes/default/template/controllers/themes/helpers/options/options.tpl
+++ b/admin/themes/default/template/controllers/themes/helpers/options/options.tpl
@@ -156,32 +156,3 @@
 	{$smarty.block.parent}
 
 {/block}
-
-
-{block name="after"}
-	<div class="panel clearfix" id="prestastore-content"></div>
-	<script type="text/javascript">
-		$.ajax({
-			type: 'POST',
-			headers: { "cache-control": "no-cache" },
-			url: 'ajax-tab.php?rand=' + new Date().getTime(),
-			async: true,
-			cache: false,
-			dataType : "html",
-			data: {
-				tab: 'AdminThemes',
-				token: '{$token|escape:'html':'UTF-8'}',
-				ajax: '1',
-				action:'getAddonsThemes',
-				page:'themes'
-			},
-			success: function(htmlData) {
-				$("#prestastore-content").html("<h3><i class='icon-picture-o'></i> {l s='Live from PrestaShop Addons!'}</h3>"+htmlData);
-			}
-		});
-
-		// These variable will move the form to another location
-		var formToMove = "appearance";
-		var formDestination = "js_theme_form_container";
-	</script>
-{/block}
diff --git a/controllers/admin/AdminThemesController.php b/controllers/admin/AdminThemesController.php
index b79afb5..3a7c66f 100644
--- a/controllers/admin/AdminThemesController.php
+++ b/controllers/admin/AdminThemesController.php
@@ -211,18 +211,7 @@ class AdminThemesControllerCore extends AdminController
                 'after_tabs' => array(
                     'cur_theme' => Theme::getThemeInfo($this->context->shop->id_theme),
                 ),
-                'submit' => array('title' => $this->l('Save')),
-                'buttons' => array(
-                    'storeLink' => array(
-                        'title' => $this->l('Visit the theme catalog'),
-                        'icon' => 'process-icon-themes',
-                        'href' => 'https://addons.prestashop.com/en/3-templates-prestashop'
-                        .'?utm_source=back-office&utm_medium=theme-button'
-                        .'&utm_campaign=back-office-'.$iso_lang_uc
-                        .'&utm_content='.(defined('_PS_HOST_MODE_') ? 'cloud' : 'download'),
-                        'js' => 'return !window.open(this.href)'
-                    )
-                )
+                'submit' => array('title' => $this->l('Save'))
             ),
         );
 
