--- a/admin/themes/default/template/controllers/login/content.tpl
+++ b/admin/themes/default/template/controllers/login/content.tpl
@@ -165,22 +165,4 @@
 		{/if}
 	</div>
 	{hook h="displayAdminLogin"}
-	<div id="login-footer">
-		<p class="text-center text-muted">
-			<a href="http://www.prestashop.com/" onclick="return !window.open(this.href);">
-				&copy; PrestaShop&#8482; 2007-{$smarty.now|date_format:"%Y"} - All rights reserved
-			</a>
-		</p>
-		<p class="text-center">
-			<a class="link-social link-twitter _blank" href="https://twitter.com/PrestaShop" title="Twitter">
-				<i class="icon-twitter"></i>
-			</a>
-			<a class="link-social link-facebook _blank" href="https://www.facebook.com/prestashop" title="Facebook">
-				<i class="icon-facebook"></i>
-			</a>
-			<a class="link-social link-github _blank" href="https://www.prestashop.com/github" title="Github">
-				<i class="icon-github"></i>
-			</a>
-		</p>
-	</div>
 </div>
