diff --git a/classes/Employee.php b/classes/Employee.php
index 71fcf965..4f14cd46 100644
--- a/classes/Employee.php
+++ b/classes/Employee.php
@@ -608,11 +608,7 @@ class EmployeeCore extends ObjectModel
      */
     public function getImage()
     {
-        if (!Validate::isLoadedObject($this)) {
             return Tools::getAdminImageUrl('prestashop-avatar.png');
-        }
-
-        return Tools::getShopProtocol() . 'profile.prestashop.com/' . urlencode($this->email) . '.jpg';
     }
 
     /**
