diff --git a/modules/blockcms/blockcms.php b/modules/blockcms/blockcms.php
index 3dbc3b7..c3c132a 100644
--- a/modules/blockcms/blockcms.php
+++ b/modules/blockcms/blockcms.php
@@ -66,7 +66,7 @@ class BlockCms extends Module
 			|| !BlockCMSModel::createTables()
 			|| !Configuration::updateValue('FOOTER_CMS', '')
 			|| !Configuration::updateValue('FOOTER_BLOCK_ACTIVATION', 1)
-			|| !Configuration::updateValue('FOOTER_POWEREDBY', 1)
+			|| !Configuration::updateValue('FOOTER_POWEREDBY', 0)
 			|| !Configuration::updateValue('FOOTER_PRICE-DROP', 1)
 			|| !Configuration::updateValue('FOOTER_NEW-PRODUCTS', 1)
 			|| !Configuration::updateValue('FOOTER_BEST-SALES', 1)
diff --git a/install/data/theme.sql b/install/data/theme.sql
index 4340262..927d028 100644
--- a/install/data/theme.sql
+++ b/install/data/theme.sql
@@ -45,7 +45,7 @@ UPDATE `PREFIX_configuration` SET value = '8' WHERE name = 'NEW_PRODUCTS_NBR';
 UPDATE `PREFIX_configuration` SET value = '10' WHERE name = 'BLOCKTAGS_NBR';
 UPDATE `PREFIX_configuration` SET value = '0_3|0_4' WHERE name = 'FOOTER_CMS';
 UPDATE `PREFIX_configuration` SET value = '0_3|0_4' WHERE name = 'FOOTER_BLOCK_ACTIVATION';
-UPDATE `PREFIX_configuration` SET value = '1' WHERE name = 'FOOTER_POWEREDBY';
+UPDATE `PREFIX_configuration` SET value = '0' WHERE name = 'FOOTER_POWEREDBY';
 UPDATE `PREFIX_configuration` SET value = 'http://www.prestashop.com' WHERE name = 'BLOCKADVERT_LINK';
 UPDATE `PREFIX_configuration` SET value = 'store.jpg' WHERE name = 'BLOCKSTORE_IMG';
 UPDATE `PREFIX_configuration` SET value = 'jpg' WHERE name = 'BLOCKADVERT_IMG_EXT';
